lib/ansible/module_utils/hpe3par.py

```diff
@@ -0,0 +1,84 @@
+# (C) Copyright 2018 Hewlett Packard Enterprise Development LP
```

Comment:
@gundalow told me recently that `module_utils` path should follow the module path, meaning that `lib/ansible/module_utils/storage/hpe3par/__init__.py` should be used. AFAIK, this change is recommended but not mandatory.

Reply:
Would be nice, and it's easier to do now before this gets released